.tree-sitter-lint.yml

```diff
@@ -1,6 +1,6 @@
 plugins:
-  replace-foo-with-bar:
-    path: ./examples/tree-sitter-lint-plugin-replace-foo-with-bar
+  replace-foo-with:
+    path: ./tests/fixtures/tree-sitter-lint-plugin-replace-foo-with
```

Comment:
For a minute I was like "wait how was this working before", the answer (which I guess makes sense) is that the information in this `plugins` section is really only used in generating the local custom binary code/dependencies (and we weren't actually generating it yet, so this was "stale" wrt the hard-coded version)